src/ucs/config/global_opts.c

```diff
@@ -210,6 +211,11 @@ static ucs_config_field_t ucs_global_opts_table[] = {
   "Enable virtual monitoring filesystem",
   ucs_offsetof(ucs_global_opts_t, vfs_enable), UCS_CONFIG_TYPE_BOOL},
 
+ {"VFS_THREAD_AFFINITY", "n",
+  "Enable affinity for virtual monitoring filesystem service thread",
```

Comment:
Enable inheriting main process affinity for virtual monitoring filesystem service thread. Setting this value to 'n' will allow the service thread to run on any CPU core.